providers/aws/components/ecs/setup.ftl

```diff
@@ -1124,129 +1124,43 @@
                                 schedule.Enabled
                     )]
 
-                    [#if networkMode == "awsvpc" ]
-
-                        [#-- Cloudfomation support not available for awsvpc network config which means that fargate isn't supported --]
-                        [#local eventRuleCliConfig =
-                            {
-                                "ScheduleExpression" : schedule.Expression,
-                                "State" : scheduleEnabled?then("ENABLED", "DISABLED")
-                            }]
-
-                        [#local eventTargetCliConfig =
-                            {
-                                "Targets" : [
-                                    {
-                                        "Id" : formatId(scheduleRuleId, "target"),
-                                        "Arn" : getExistingReference(ecsId, ARN_ATTRIBUTE_TYPE),
-                                        "EcsParameters" : {
-                                            "TaskCount" : schedule.TaskCount
-                                        } +
-                                        attributeIfTrue(
-                                            "NetworkConfiguration",
-                                            networkMode = "awsvpc",
-                                            {
-                                                "awsvpcConfiguration" : {
-                                                    "Subnets" : subnets,
-                                                    "AssignPublicIp" : publicRouteTable?then("ENABLE", "DISABLED")
-                                                }
-                                            }
-                                        ) +
-                                        attributeIfTrue(
-                                            "LaunchType",
-                                            engine == "fargate",
-                                            "FARGATE"
-                                        )
```

Comment:
I think this needs to be moved over to
 https://github.com/codeontap/gen3/pull/1009/files#diff-48e25ae413e06c838128cd60d543fcdfR1147 

The network configuration still needs to be applied for fargate and awsvpc

Reply:
@roleyfoley , fixed